services/mirror/mirror_pull.go

```diff
@@ -549,6 +549,14 @@ func SyncPullMirror(ctx context.Context, repoID int64) bool {
 		return false
 	}
 
+	// Update License
+	if err = repo_module.AddRepoToLicenseUpdaterQueue(&repo_module.LicenseUpdaterOptions{
```

Comment:
Only for default branch?

Reply:
I think sync mirror always syncs default branch, doesn't it?